piptools/sync.py

```diff
@@ -13,6 +13,7 @@
     'pip-review',
     'setuptools',
     'wheel',
+    'pkg-resources',
```

Comment:
pkg-resources does not seem to be a pypi package, I believe this should be removed from the PR because it is also irrelevant to the change.